classes/patterns/class-pageheader.php

```diff
@@ -38,10 +38,11 @@ public static function get_name(): string {
 	 * @param array $params Optional array of parameters for the config.
 	 */
 	public static function get_config( $params = [] ): array {
-		$classname  = 'is-pattern-p4-page-header';
-		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
-		$pos_class  = 'left' === static::$media_position ? ''
+		$classname         = 'is-pattern-p4-page-header';
+		$media_link        = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+		$pos_class         = 'left' === static::$media_position ? ''
 			: 'has-media-on-the-' . static::$media_position;
+		$title_placeholder = $params['title_placeholder'] ?? '';
```

Comment:
To be more precise, it's the title, or title content, not the placeholder

Reply:
True, true, but it is used as a placeholder since it's meant to be changed 🤔

Reply:
:thinking:  I see your point, mine was just to not confuse it with the `placeholder` attribute, but that works too :+1: